library/component/card/card.css

```diff
@@ -0,0 +1,116 @@
+.card{
+    width:23rem;
+    min-height:30rem;
+    text-align:left;
+    border:2px solid var(--primary-color);
```

Comment:
For better readability.

property:**(space)**value

Reply:
I will update it , thanks Varun